exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/text/compliant/FieldVarCharOutput.java

```diff
@@ -45,11 +45,11 @@
   static final String COL_NAME = "columns";
 
   // array of output vector
-  private final VarCharVector [] vectors;
+  private final NullableVarCharVector [] vectors;
```

Comment:
Let's think about this. CSV is defined as a fixed set of columns. For this reason, the original code used non-nullable values for each column.

The question would be, what about this fix means that defined columns should now become null? Are there other solutions to this bug?

Changing the column from required to nullable can have impact on the user experience and on the client: where code once expected required columns, now that code must change to handle nullable types.